### PR TITLE
tweak(toast): Update npm repo & homepage URLs

### PR DESCRIPTION
**Pull request by replygirl**

### CODE REVIEW

File: packages/toast/package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "@nuxtjs/toast",
-  "version": "3.3.1",
+  "version": "3.3.2",
```
Comment (danielroe) [inline packages/toast/package.json:3]:
```suggestion
  "version": "3.3.1",
```

**Review by danielroe — COMMENTED** [review]
Looks good. But the version shouldn't be updated; we'll do that when we release.

**Comment by replygirl** [comment]
> Looks good. But the version shouldn't be updated; we'll do that when we release.

Gotcha. I thought I had to pump to pass the status check, but I guess I just needed a dummy commit so there was something to squash

**Review by danielroe — APPROVED** [review]
Thank you ❤️

**Pull request merged**